docs/markdown/overview.md

```diff
@@ -0,0 +1,67 @@
+# Interledger Overview
+
+Interledger enables the seamless exchange of value across different payment networks. Traditional payment networks 
+operate independently from each other. Sending value is easy only if the sender and recipient have accounts on the same 
+network, but it can be slow and expensive if they have accounts on different networks. Interledger makes it easy to 
+transact in whatever currency or payment network you choose because it is not tied to any one company, blockchain, or 
+currency. Using Interledger, you can easily send XRP to someone who wants to receive ETH, or you can send USD to someone 
+who wants to receive EUR.
+
+
+## What is Interledger?
+Interledger is a network of computers that enables the sending of value across independent payment networks. 
+Similar to how the Internet routes packets information, Interledger routes packets of value. Computers on the Interledger 
+network are called *nodes*. Nodes can take one or more of the following roles: 
+* Sender – Initiates a value transfer.
+* Router – Applies currency exchange and forwards packets of value. This is an intermediary node between the sender 
+and the receiver. 
+* Receiver – Receives the value.
+
+![ILP-nodes](../images/ilp-nodes.png)
```

Comment:
Can we show a transfer that doesn't include XRP? Otherwise it might contribute to the misconception the XRP is required for / otherwise tied to Interledger

Reply:
Let's remove `XRP` from _one_ of the examples, instead of adding a 3rd example.

Reply:
Does this work?
https://github.com/pgarg-ripple/interledger.github.io/blob/pg-overview/docs/images/ilp-packets-2.png